src/dlsnode/storage/BucketFile.d

```diff
@@ -570,6 +612,47 @@ public class BucketFile: OutputStream
         }
     }
 
+    private bool readBucketHeaderAsync(JobNotification suspended_job)
+    {
+        BucketHeader header;
+
+        // rewind to the beginning of the file
+        this.seek(0);
```

Comment:
This covers only the case when reading from the bucket, I need to cover the case when appending to the existing file.

Reply:
OK, this is addressed - https://github.com/sociomantic-tsunami/dlsnode/pull/35/files#diff-87f86921c10577135fe76b018eec90a8R379